client/chat/channel-info-card.tsx

```diff
@@ -101,34 +144,21 @@ export function ConnectedChannelInfoCard({
     )
   })
 
-  const isChannelNotFound =
-    isFetchError(findChannelError) && findChannelError.code === ChatServiceErrorCode.ChannelNotFound
   const isUserBanned =
     isFetchError(joinChannelError) && joinChannelError.code === ChatServiceErrorCode.UserBanned
 
-  const icon =
-    channelInfo && (!channelInfo.private || isUserInChannel) ? (
-      <StyledChannelIcon as={ChannelIcon} />
-    ) : (
-      <ErrorChannelIcon icon='warning' size={56} />
-    )
-
-  let subtitle
-  if (isChannelNotFound) {
-    subtitle = (
-      <ErrorText>
-        This channel could not be found. It might not exist, or it may have been re-created by
-        someone else.
-      </ErrorText>
-    )
-  } else if (channelInfo?.private && !isUserInChannel) {
-    subtitle = <ErrorText>This channel is private and requires an invite to join.</ErrorText>
-  } else if (isUserBanned) {
-    subtitle = <ErrorText>You are banned from this channel.</ErrorText>
-  } else if (channelInfo?.userCount) {
-    subtitle = (
-      <Body1>{`${channelInfo.userCount} member${channelInfo.userCount > 1 ? 's' : ''}`}</Body1>
+  let descriptionText
+  if (channelInfo?.private && !isUserInChannel) {
+    descriptionText = (
+      <NoChannelDescriptionText>
+        <PrivateChannelIcon />
+        This channel is private and requires an invite to join.
+      </NoChannelDescriptionText>
     )
+  } else if (channelInfo?.description) {
+    descriptionText = channelInfo.description
+  } else {
+    descriptionText = <NoChannelDescriptionText>No channel description.</NoChannelDescriptionText>
```

Comment:
This doesn't feel like a very user-focused message, maybe "No description provided." or "This channel has no description." (which is similar in structure to your private message)